Bio/Blast/NCBIWWW.py

```
@@ -31,9 +38,20 @@
 from urllib.request import Request
 from urllib.request import urlopen
 
+from Bio import BiopythonDeprecationWarning
 from Bio import BiopythonWarning
 from Bio._utils import function_with_previous
 
+warnings.warn(
```

Ruff (0.16.1)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)